provider/cmd/pulumi-resource-nuage/nuage_provider/serverless_database.py

```diff
@@ -172,82 +184,19 @@ def __init__(
             )
         )
 
-        if args.data_api:
-            # Secret containing DB credentials
-            secret = aws.secretsmanager.Secret(
-                resource_name=f"{args.resource_name}-secret",
-                name=pulumi.Output.all(
-                    cluster_identifier=cluster.cluster_identifier,
-                    database_name=cluster.database_name,
-                ).apply(
-                    lambda args: "/".join(
-                        [
-                            "rds-db-credentials",
-                            args["cluster_identifier"],
-                            args["database_name"],
-                        ]
-                    )
+        if args.bastion_enabled and args.bastion_subnet_id:
+            bastion = Bastion(
+                f"{resource_name}/bastion",
+                args=BastionArgs(
+                    name=self.name_.apply(lambda name: f"{name}-bastion"),
```

Comment:
The "-bastion" part would end up after the random suffix... not great

Is there a way to do it differently ?